ffv1.md

```diff
@@ -287,24 +287,36 @@ For each plane of each slice, a Quantization Table Set is selected from an index
 
 Background: in first implementations of FFV1 bitstream, the index for Cb and Cr planes was stored even if it is not used (chroma_planes set to 0), this index is kept for version <= 3 in order to keep compatibility with FFV1 bitstreams in the wild.
 
-## Color space
+## Color spaces
 
-FFV1 supports two color spaces: YCbCr and JPEG2000-RCT. Both color spaces allow an optional Alpha plane that can be used to code transparency data.
+FFV1 can losslessly compress two color spaces: YCbCr and RGB. Both color spaces allow an optional Alpha plane that can be used to code transparency data.
 
-### YCbCr
+## Components
 
-In YCbCr color space, the Cb and Cr planes are optional, but if used then MUST be used together. Omitting the Cb and Cr planes codes the frames in grayscale without color data. An FFV1 `Frame` using YCbCr MUST use one of the following arrangements:
+FFV1 encodes 1 Luma component, optionally 2 Chroma components, and optionally 1 Alpha component.
+
+The Cb and Cr planes are optional, but if used then MUST be used together. Omitting the Cb and Cr planes codes the frames in grayscale without color data. An FFV1 `Frame` using YCbCr MUST use one of the following arrangements:
 
 - Y
 - Y, Alpha
 - Y, Cb, Cr
 - Y, Cb, Cr, Alpha
 
-When FFV1 uses the YCbCr color space, the Y plane MUST be coded first. If the Cb and Cr planes are used then they MUST be coded after the Y plane. If an Alpha (transparency) plane is used, then it MUST be coded last.
+The Y plane MUST be coded first. If the Cb and Cr planes are used then they MUST be coded after the Y plane. If an Alpha (transparency) plane is used, then it MUST be coded last.
+
+## Pixel transformations
+
+Pixel transformations permit to modify the pixel data in a reversible way in order to improve the compression ratio, before being encoded. An example of usage is to compress RGB content which could have each component independently compressed but with a less optimal compression ratio.
+
+Pixel transformation is indicated in the bitstream in order to permit the decoder to configure the output accordingly to encoder input.
+
+### No Pixel transformation
+
+There is no Pixel transformation, the source planes are directly compressed.
 
 ### JPEG2000-RCT
 
-JPEG2000-RCT is a Reversible Color Transform that codes RGB (red, green, blue) planes losslessly in a modified YCbCr color space. Reversible conversions between YCbCr and RGB use the following formulae.
+JPEG2000-RCT is a Reversible Color Transform that codes RGB (red, green, blue) planes losslessly in a modified YCbCr color space. Reversible Pixel transformations between YCbCr and RGB use the following formulae.
```

Comment:
`pixel` instead of `Pixel`?

Reply:
Following [this comment about titlecase for defined word](https://github.com/FFmpeg/FFV1/pull/92#discussion_r159676822)